web/src/main/java/com/marklogic/hub/web/web/CurrentProjectController.java

```diff
@@ -230,15 +230,15 @@ public String preInstallCheck() {
     }
 
     private void startProjectWatcher() throws IOException {
-        String pluginDir = hubConfig.getHubPluginsDir().toString();
         String flowsDir = hubConfig.getFlowsDir().toString();
         String stepDefinitionsDir = hubConfig.getStepDefinitionsDir().toString();
         String entitiesDir = hubConfig.getHubEntitiesDir().toString();
+        String mappingsDir = hubConfig.getHubMappingsDir().toString();
         if (!watcherService.hasListener(this)) {
-            watcherService.watch(pluginDir);
             watcherService.watch(flowsDir);
             watcherService.watch(stepDefinitionsDir);
             watcherService.watch(entitiesDir);
+            watcherService.watch(mappingsDir);
```

Comment:
Now that we are watching the entire mapping directory, should we remove the watch service from the mappingManagerService for the individual mapping folder name?

Reply:
We can leave them there now. Let us test after beta release.